### ISSUE TITLE
tp.user don't show autocomplete and also gives a message on the console

### ISSUE THREAD

**Issue by reisnobre** [SilentVoid13/Templater#1159]
**Plugin information (please complete the following information):**

-   OS: [macOS Ventura 13.5]
-   Templater version: [1.16]
-   Obsidian version: [1.4.2]
-   Templater settings:
    - Scripts folder: /Meta/Scripts/
    -  Everything else is pretty standard

**Describe the bug**
When typing tp.user. I don't get any autocomplete information and on the log I get the following message
```
pugin:templater-obsidian:4110 Uncaught TypeError: Cannot read properties of undefined (reading 'functions')
    at Documentation.get_all_functions_documentation (plugin:templater-obsidian:4110:45)
    at Autocomplete.getSuggestions (plugin:templater-obsidian:4168:40)
    at t.trigger (app.js:1:1440484)
    at e.trigger (app.js:1:1868231)
    at app.js:1:1254961
    at l (app.js:1:251148)
    at c (app.js:1:251264)
```

**Expected behavior**
I see the list of available functions defined on my script folder 
**Screenshots**
![Screenshot 2023-08-05 at 19 00 15](https://github.com/SilentVoid13/Templater/assets/12832066/09e93e43-5148-4873-a4f6-954768ef5466)

As you can see, the scripts are being detected correctly.

**Additional context**
Just using the function and not relying on the autocomplete results on the expected behaviour, so I think the scripts themselves are fine, since templater is able to use them and gives the correct values.

**Comment by reisnobre** [comment]
Just confirmed that for some reason,  at the `get_all_functions_documentation` the `this.documentation.tp` doesn't have a user value for some reason

**Comment by Zachatoo** [comment]
Should be fixed with #980, marking as closed
